### ISSUE TITLE
The Flutter team should be directly involved into most proiminent packages

### ISSUE THREAD

**Issue by feinstein** [flutter/flutter#39771]
@filiph as we discussed, [flutter_facebook_plugin](https://github.com/roughike/flutter_facebook_login) hasn't been updated in a while and the issues and PRs are pilling up. Since this is such an important project, it's important for Flutter's own good image that it doesn't seems abandoned.

IMHO I think it would be best if the Flutter team assembled some people responsible for tracking the most important Flutter packages and approached the maintainers, suggesting that a person from Google be inserted into the project's repository. This way the Flutter ecosystem will be much more resilient, having much less chances to suffer from the same problems that Javascript had, with high-profile repos just being abandoned. This would transmit an even better image to the market in general.

**Comment by feinstein** [comment]
@filiph I just had an idea that could help on this. Dart pub has a score for each package, so packages with only one maintainer on their github repository should get a heavy penalty on their score, as this is a sign of risk.

**Comment by Hixie** [comment]
cc @csells

**Comment by csells** [comment]
I like this idea, although we want to be careful not to penalize high quality, well-maintained packages simply because they came from a single author if that author is responsive.

cc @mit-mit

**Comment by feinstein** [comment]
@csells, even if the author is responsive he's still human, so there is always the resilience factor. I don't like talking about this, but professionally we have to consider what happens if an author dies, leaving a popular and great package behind.

**Comment by timsneath** [comment]
Unfortunately, the team just doesn't have the scale to participate in third-party plug-in development. Even in large projects, we have to focus and we've chosen to put our development resources primarily into the core framework. However, the good news is that since this issue has been opened we've established the [Flutter Favorites](https://flutter.dev/docs/development/packages-and-plugins/favorites) program (thanks to @csells), which provides a governance body for highlighting plugins that have the kind of resilience that you describe. Our hope is that this will grow over time. 

Closing this issue for now. @mit-mit, I suspect that the new package publisher field makes it hard to identify whether there's a single author or multiple authors behind a package, but I'll let you open a fresh issue if you want to incorporate development team size into the pub.dev score rating.
